src/Service.Host/package.json

```diff
@@ -18,7 +18,7 @@
         "collectCoverageFrom": [
             "client/src/**/*.{js}"
         ],
-        "collectCoverage": true,
+        "collectCoverage": false,
```

Comment:
did you mean to switch coverage off?

Reply:
Was doing that in testing, forgot to turn it back on...